### ISSUE TITLE
[Feature request] Add support for triggering python scripts

### ISSUE THREAD

**Issue by atrus05** [AShifter/harmony-span#14]
This project is for me a dream come true (as mentioned before), and I see lots of use-cases for me if calling scripts was added to the feature list.

I want to be able to press a button on the Harmony remote control (I am using the Harmony Companion) which triggers a python script of my choosing which resides on my server. I have a couple of scripts which does things a lot better than the Harmony "activity" I have set up can do for example. And some of my scripts does things that the Harmony ecosystem can't handle at all.

In short, being able to trigger python scripts from Harmony-Span would be mind-blowingly exciting :)

**Comment by AShifter** [comment]
Thanks for opening an issue!

This probably isn't something I'll be implementing as it does lie outside the scope of the project - though it should be pretty easy to build a RESTful API into your own applications that HarmonySpan could trigger for you. It looks like the Flask library could be a big help here. Let me know if you need any help with implementation - I'll keep this issue open for now and I'm more than happy to help!

**Comment by miketheman986** [comment]
@atrus05, did you have any luck with this?  I'm looking to do something similar, but not really sure where to start.
